src/Conference/UpdateConference.jsx

```diff
@@ -1,12 +1,10 @@
 import React from "react";
-import Input from "../Input";
 import { useHistory } from "react-router-dom";
+import { withCookies, useCookies } from "react-cookie";
```

Comment:
You're using a hook `useCookies` and a high order component `withCookies` in this component. You only need to use one